public/app/features/dashboard-scene/serialization/testfiles/panel_repeats.json

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
@kaydelaney we already have a repeating rows and panels dashboard (in this testfiles folder ), imported in the test ts file as repeatingRowsAndPanelsDashboardJson